site/docs/components/_index/components-list.tsx

```diff
@@ -150,8 +150,8 @@ export const componentDetails: ComponentDetails[] = [
     name: "Avatar",
     description:
       "Represents a person. It has three variants to account for different types of data: photo, initials and a default version when neither is available.",
-    devStatus: ComponentStatus.IN_BACKLOG,
-    designStatus: ComponentStatus.IN_BACKLOG,
+    devStatus: ComponentStatus.IN_PROGRESS,
```

Comment:
@Fercas123 i'm not sure if this is ready now or in progress, feel free to change :)